app/app.ts

```diff
@@ -26,10 +25,18 @@ import VeeValidate from 'vee-validate';
 import ChildWindow from 'components/windows/ChildWindow.vue';
 
 const { ipcRenderer, remote } = electron;
-
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 const isProduction = process.env.NODE_ENV === 'production';
 
+window['obs'] = window['require']('obs-studio-node');
+
+{ // Set up things for IPC
+  // Connect to the IPC Server
+  window['obs'].IPC.connect(process.env.SLOBS_IPC_PATH);
```

Comment:
Theoretically our child windows shouldn't need connections to the OBS server.  That said there may be a spot or two where we accidentally call directly into OBS from the child window.  If you move this into `app/services/app.ts` (so only the main window has a connection) do things start breaking?

Reply:
Also given that this environment variable is set in the main process, I don't think you can guarantee that it will always be here this early.  It's safer to use `electron.remote.process.env` instead I think.

Reply:
> If you move this into app/services/app.ts (so only the main window has a connection) do things start breaking?

Yes, because the child window has no connection to the obs ipc server. I did not write a layer that talks to the main window renderer for this, instead I have the child windows also connect to the ipc server.

This should not have any negative impact at all, it should actually improve things. In the ideal case, I want to avoid having to block any other v8 context other than the one the command is ran on. OBS handles almost all synchronization needs, so there is not much to worry about there.

> It's safer to use electron.remote.process.env instead I think.

You are correct, I will fix that.